TrafficCapture/dockerSolution/src/main/docker/migrationConsole/cluster_tools/src/cluster_tools/base/main.py

```diff
@@ -4,41 +4,37 @@
 import argcomplete
 from console_link.environment import Environment
 import logging
-import sys
+from datetime import datetime
 
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.INFO)
-
-
-class ClusterToolsFilter(logging.Filter):
-    def filter(self, record):
-        return record.name.startswith('cluster_tools') or record.name.startswith('tools')
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
 
+logger = logging.getLogger(__name__)
 
-def setup_logging():
-    """Sets up logging with a file handler for all logs and a stdout handler for 'cluster_tools' logs only."""
 
-    from datetime import datetime
+def setup_file_logging(tool_name: str):
+    """Sets up logging with a file handler for all logs."""
     timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
     logs_dir = os.getenv("SHARED_LOGS_DIR_PATH", "./logs")
-    log_file_path = os.path.join(logs_dir, f'cluster_tools/log_{timestamp}_{os.getpid()}.log')
+    host_name = os.getenv("HOSTNAME", "localhost")
```

Comment:
not a note for this PR really, but do you know if we follow this same convention of falling back to `localhost` for all of our tools that use this logging directory?